src/storage/src/mem_table.rs

```diff
@@ -196,6 +222,26 @@ impl MemTable {
     {
         self.buffer.range(key_range)
     }
+
+    fn calculate_origin_size(&mut self, origin_value: Option<KeyOp>, key_len: usize) {
+        if let Some(origin_value) = origin_value {
```

Comment:
Maybe use the name `sub_origin_size` since this func does more than calculation.